medusa/server/web/config/handler.py

```diff
@@ -46,7 +46,7 @@ def index(self):
         """
         Render the Help & Info page
         """
-        t = PageTemplate(rh=self, filename='config.mako')
+        t = PageTemplate(rh=self, filename='index.mako')
 
         try:
```

Comment:
This route can be simplified now.
Many of the values are already served from the API.

The route function can be:
```py
    def index(self):
        """
        Render the Help & Info page.

        [Converted to VueRouter]
        """
        t = PageTemplate(rh=self, filename='index.mako')
        return t.render(submenu=self.ConfigMenu(), topmenu='config')
```

Reply:
Oh yeah, and also remove the unused imports at the top of the file.